File: app/src/main/java/in/sudopk/vaishnavacalendar/calendar/CalendarDelegate.java
package in.sudopk.vaishnavacalendar.calendar;

import in.sudopk.vaishnavacalendar.VCalendar;

public interface CalendarDelegate {
    void onRefresh();

    void tryShowingCalendar();

    void onChangeLocationRequest();

    void onCalendarResponse(VCalendar body);

    void onCalendarRequestFailed();
}

File: app/src/main/java/in/sudopk/vaishnavacalendar/calendar/CalendarDelegate.kt
package `in`.sudopk.vaishnavacalendar.calendar

import `in`.sudopk.vaishnavacalendar.VCalendar

interface CalendarDelegate {
    fun onRefresh()

    fun tryShowingCalendar()

    fun onChangeLocationRequest()

    fun onCalendarResponse(body: VCalendar)

    fun onCalendarRequestFailed()
}
